Autopilot/AttitudeManager/InputCapture.c

```diff
@@ -285,6 +296,7 @@ void __attribute__((__interrupt__, no_auto_psv)) _IC7Interrupt(void)
     } else {
         end_time[6] = IC7BUF;
         new_data_available[6] = 1;
+        last_capture_time = getTime(); //Capture the time. Used for detecting disconnects
     }
```

Comment:
So the problem with this, is even if your receiver disconnects,  it outputs either the last known settings or it will output the failsafe signal (which can be programmed into the controller). 

This will only work if you physically pull the PWM cable.